.github/workflows/misc-migtests.yml

```diff
@@ -85,15 +92,15 @@ jobs:
           GCS_REFRESH_TOKEN: ${{ secrets.PGUPTA_GCS_REFRESH_TOKEN }}
           AWS_ACCESS_KEY_ID: ${{ secrets.RAHULB_S3_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.RAHULB_S3_SECRET_ACCESS_KEY }}
-        if: always()
+        if: ${{ !cancelled() }}
         run: |
           versions=("2.20.5.0-b72" "2.21.1.0-b271" "2024.1.1.0-b137")
           for version in "${versions[@]}"; do
             echo "Running tests on version $version"
 
             echo "Start YugabyteDB cluster"
-            docker pull yugabytedb/yugabyte:$version
-            VERSION=$version docker compose -f migtests/setup/yb-docker-compose.yaml up -d
+            docker run -d --name yugabytedb-$version -p7000:7000 -p9000:9000 -p15433:15433 -p5433:5433 -p9042:9042 \
+            yugabytedb/yugabyte:$version bin/yugabyted start --background=false --ui=false
```

Comment:
nit: Can we format this as well like the other files.

Reply:
done